config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,10 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    provider: "AWS",
+    aws_access_key_id: ENV["PRODUCTION_AWS_ACCESS_KEY_ID"] || "",
+    aws_secret_access_key: ENV["PRODUCTION_AWS_SECRET_KEY"] || ""
+  }
+  config.fog_directory  = ENV["PRODUCTION_AWS_BUCKET"] || ""
+  config.fog_public     = false                                        # optional, defaults to true
```

Comment:
You probably want this to be true otherwise users won't be able to access the images without your credentials.